.gitignore

```diff
@@ -5,6 +5,8 @@ coverage
 node_modules
 dist
 lat-lng-generation-report.txt
+arbo-lat-lng-generation-report.txt
+sars-cov-2-lat-lng-generation-report.txt
```

Comment:
Lots of this PR is just copy-paste from the arbo ETL. I know there's fields missing (Ex. `unRegion`, `whoRegion`, `assay`, etc.) and I'm going to make tickets for those fields but this is just to get something standing so that we can iterate on it.